R/plotDataSetList.R

```diff
@@ -1245,6 +1298,15 @@ Plot.Parameters.DataSetList <- function(dsList, f_min = NULL, f_max = NULL,
                               name = alg,
                               legendgroup = ~algId,
                               showlegend = showlegend)
+      p[[j]] %<>%
```

Comment:
we just want to adjust some layouts here, right?

Reply:
Yes, we want to make sure all axis have a label